diagnosis/getkuberneteslogs.sh

```diff
@@ -191,6 +165,27 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+#checks if azure-cli is installed   
+requirements
+
+#workaround for SSL interception
+export AZURE_CLI_DISABLE_CONNECTION_VERIFICATION=1 
+export ADAL_PYTHON_SSL_NO_VERIFY=1
+
+#Validate resource-group
+location=$(az group show -n $RESOURCE_GROUP --query location)
+if [ $? -ne 0 ]; then
```

Comment:
az commands always return exit code 0. Check for `$?` is not a good test in this case.

Reply:
Nope, if the resource group is not found the exit code is 3.

Reply:
you are right, sorry for that